eslint-plugin/src/index.ts

```diff
@@ -0,0 +1,56 @@
+
+import { TSDocParser } from "@microsoft/tsdoc";
+import { allTsdocMessageIds } from "@microsoft/tsdoc/lib/parser/TSDocMessageId";
+import * as eslint from "eslint";
+import * as ESTree from "estree";
+
+const messageIds: {[x: string]: string} = {};
+
+allTsdocMessageIds.forEach((messageId: string) => {
+    messageIds[messageId] = messageId + ": {{ unformattedText }}";
```

Comment:
You can use a template string here:

```suggestion
  messageIds[messageId] = `${messageId}: {{ unformattedText }}`;
```